ui/selectors/assets.ts

```diff
@@ -91,6 +95,10 @@ export type BalanceCalculationState = {
       conversionRates?: Record<string, unknown>;
       historicalPrices?: Record<string, unknown>;
       networkConfigurationsByChainId?: Record<string, unknown>;
+      accountsByChainId?: Record<
+        string,
+        Record<string, { balance: string; address: string }>
+      >;
```

Comment:
Updating `BalanceCalculationState` type with partial accountsByChainId type to be used with `selectAccountsByChainIdForBalances`